plugins/tests/pandera/test_wf.py

```diff
@@ -0,0 +1,74 @@
+import typing
+
+import pandas
+import pandera
+import pytest
+
+import flytekitplugins.pandera
```

Comment:
does this line pass lint?  it's not needed in code right?  it's just needed to trigger everything.

Reply:
we actually have the same problem with the regular pandas library.  Tasks and workflows that take in and return pandas dataframes need to trigger the SchemaTransformer, but unless they explicitly import flytekit.types.schema, the transformer isn't registered and it doesn't run.  Can you think of a way around this?